### ISSUE TITLE
missing something

### ISSUE THREAD

**Issue by Rickyc81** [aedoardo/telegram-twitch-sub-bot#3]
Hi, Is it possible that you have forgotten something?

in the function:

```    def _loadCommands(self, dp):
        commandsCollection = self._mongo["streamers-telegram-groups"]["botcommands"]
        commandsClass = Commands(self._mongo, self._clientID)
        messagesClass = Messages(self._mongo, self._clientID)

        cursor = commandsCollection.find({})
        print(cursor)
        for command in cursor:
            self._commands.append(command["name"])
            if command["type"] == "command":
                dp.add_handler(CommandHandler(command["name"], getattr(commandsClass, command["callback"]),
                                              run_async=command["async"]))
            elif command["type"] == "message":
                dp.add_handler(
                    MessageHandler(getattr(Filters, command["filters"]), getattr(messagesClass, command["callback"])))
            elif command["type"] == "buttonCommand":
                dp.add_handler(
                    CallbackQueryHandler(getattr(commandsClass, command["callback"]))
                )

        print("COMMANDS & MESSAGES LOADED {}".format(len(self._commands)))
```
that return: 
COMMANDS & MESSAGES LOADED 0
there is nowhere written that commands must read from the database, you do not load them too and they are not the commands I see in the attached .json file

I don't even find much correspondence in the attached .json file, apart from "name" and "callback". but the structure?

can you clarify this thing? Thanks

**Comment by Rickyc81** [comment]
then I see that in the commands.py file it loads the commands from the .json file ..

it is not at all clear how what you wrote works ..
forgive me, maybe it would have been more logical to keep the repo private and let it come out calmly, complete and well documented

**Comment by aedoardo** [comment]
Hello @Rickyc81 , I forgot to add the `botcommands` collection. I added it to the repository. The `json` file is used to build and to handle the different menus. The `botcommands` collection, instead, adds the handlers to the bot (and some of them are actually deprecated, I think).

Do not worry, I released this code just to give a base idea on how a bot for subscriber on a Twitch channel could work. I don't touch the code since 1 year, and I'm pretty sure that the APIs (Twitch & Telegram) will be deprecated in a while.
